psi4/src/psi4/libpsipcm/CMakeLists.txt

```diff
@@ -6,5 +6,7 @@ if(TARGET PCMSolver::pcm)
   target_link_libraries(psipcm
     PUBLIC
       PCMSolver::pcm
+      mints
```

Comment:
Elaborate on why this is needed?

Reply:
I think the note below about pulling in potentialint.h and l2/engine.h in turn explains the additional targets here. iirc, there's a lot of missing `mints` deps to directories.

Reply:
Yup, we're using mints classes here, so we should depend on them.